hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/transactionhandler/AbstractAllowanceTransactionHandler.java

```diff
@@ -60,7 +61,7 @@ public void updateTransaction(Transaction transaction, RecordItem recordItem) {
             var approvedForAll = nftApproval.hasApprovedForAll() && nftApproval.getApprovedForAll().getValue();
             NftAllowance nftAllowance = new NftAllowance();
             nftAllowance.setApprovedForAll(approvedForAll);
-            nftAllowance.setOwner(EntityId.of(nftApproval.getOwner()).getId());
+            nftAllowance.setOwner(getOwner(nftApproval.getOwner(), payerAccountId));
```

Comment:
nit: Might consider taking advantage of the existing fallback logic in `EntityIdService` and to prepare for alias everywhere.

```suggestion
            nftAllowance.setOwner(entityIdService.lookup(nftApproval.getOwner(), payerAccountId));
```